packages/legend-application-studio/src/components/editor/edit-panel/service-editor/__tests__/ServiceRegistrationTest.test.tsx

```diff
@@ -301,14 +300,6 @@ test(
       );
     const service = serviceEditorState.service;
     expect(service.owners).toHaveLength(2);
-    fireEvent.click(getByTitle(editPanel, 'Add owner'));
```

Comment:
what would it take to readd this part of the test with the new flow. Is it expensive/worth it?

Reply:
The steps would be:

     1. fireEvent -> target the dropdown 
     2. fireEvent -> change the input 
     3. We need to mock the data in the dropdown because the input will trigger an API call
     4. fireEvent -> choose one from the dropdown
   

I could try if it's worth it.